Dockerfiles/testrail_apache/Dockerfile

```diff
@@ -29,7 +28,7 @@ RUN \
 RUN \
     docker-php-ext-install mbstring && \
     if [ "x$ARG_PHP_VERSION" = "5."* ] ; then docker-php-ext-install mysql; \
-    else docker-php-ext-install mysqli; \
+    else docker-php-ext-install mysqli pdo pdo_mysql; \
```

Comment:
I think this was already done in https://github.com/gurock/testrail-docker/commit/8674d81ef634a4cd97adebd56ec463c6aa62d6e9